waifus/madoka yabuuchi

```diff
@@ -0,0 +1,11 @@
+{
+	"type" : 0,
+	"english-name" : "Madoka yabuuchi",
+	"image" : "https://cdn.myanimelist.net/images/characters/8/379321.jpg",
+	"japanese-name" : "藪内 円",
+	"year" : 2019,
```

Comment:
Where does this number come from? You need to say where this number comes from in the notes. Here's an example of some notes https://github.com/yourWaifu/is-your-waifu-legal/blob/92ea548cf578e66dc4f61f7e283b1bb267507cf3/waifus/sucy%20manbavaran.json#L10-L12